src/app/components/ScrollablePromo/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,438 @@
+    margin-left: 20%;
+  }
+}
+
+@media (min-width: 80rem) {
+  .emotion-0 {
+    margin-left: 40%;
+  }
+}
+
+@supports (display: grid) {
+  .emotion-0 {
+    display: block;
+    width: initial;
+    margin: 0;
+  }
+
+  @media (max-width: 14.9375rem) {
+    .emotion-0 {
+      grid-template-columns: repeat(6, 1fr);
+      grid-column-end: span 6;
+      grid-column-start: 1;
+    }
+  }
+
+  @media (min-width: 15rem) and (max-width: 24.9375rem) {
+    .emotion-0 {
+      grid-template-columns: repeat(6, 1fr);
+      grid-column-end: span 6;
+      grid-column-start: 1;
+    }
+  }
+
+  @media (min-width: 25rem) and (max-width: 37.4375rem) {
+    .emotion-0 {
+      grid-template-columns: repeat(6, 1fr);
+      grid-column-end: span 6;
+      grid-column-start: 1;
+    }
+  }
+
+  @media (min-width: 37.5rem) and (max-width: 62.9375rem) {
+    .emotion-0 {
+      grid-template-columns: repeat(5, 1fr);
+      grid-column-end: span 5;
+      grid-column-start: 1;
+    }
+  }
+
+  @media (min-width: 63rem) and (max-width: 79.9375rem) {
+    .emotion-0 {
+      grid-template-columns: repeat(5, 1fr);
+      grid-column-end: span 5;
+      grid-column-start: 2;
+    }
+  }
+
+  @media (min-width: 80rem) {
+    .emotion-0 {
+      grid-template-columns: repeat(10, 1fr);
+      grid-column-end: span 10;
+      grid-column-start: 5;
+    }
+  }
+}
+
+.emotion-2 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  overflow-wrap: break-word;
+  list-style: none;
+  padding-right: 0;
+  margin: 0;
+  overflow-x: scroll;
+  scroll-behavior: auto;
+  -webkit-overflow-scrolling: touch;
+  scrollbar-width: none;
+  -ms-overflow-style: none;
+}
+
+.emotion-2::-webkit-scrollbar {
+  display: none;
+}
+
+.emotion-4 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-shrink: 0;
+  -ms-flex-negative: 0;
+  flex-shrink: 0;
+}
+
+@media (min-width: 0rem) {
+  .emotion-4 {
+    margin-right: 0.5rem;
+  }
+
+  .emotion-4:first-child {
+    margin-right: 0.5rem;
+  }
+
+  .emotion-4:last-child {
+    margin-left: 0.5rem;
+  }
+}
+
+@media (min-width: 25rem) {
+  .emotion-4 {
+    margin-right: 1rem;
+  }
+
+  .emotion-4:first-child {
+    margin-right: 1rem;
+  }
+}
+
+@media (min-width: 63rem) {
+  .emotion-4 {
+    margin-right: 1rem;
+  }
+
+  .emotion-4:first-child {
+    margin-right: 0;
+  }
+}
+
+.emotion-6 {
+  margin-bottom: 1.5rem;
+  background-color: #FFFFFF;
+  padding: 1rem;
+}
+
+@media (min-width: 0rem) {
+  .emotion-6 {
+    width: 14.8125rem;
+  }
+}
+
+@media (min-width: 37.5rem) {
+  .emotion-6 {
+    width: 11.125rem;
+  }
+}
+
+@media (min-width: 63rem) {
+  .emotion-6 {
+    width: 12.6875rem;
+  }
+}
+
+.emotion-8 {
+  width: 100%;
+  text-overflow: ellipsis;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+  color: #202224;
```

Comment:
Is this the link colour?

Reply:
Thought we used #222222 for links?

Reply:
Design in Zeplin said to use #202224. I am currently double checking with Gail

Reply:
#202224 is a new chameleon colour for links, this will be slowly replacing #222222